1 table.sql

```diff
@@ -0,0 +1,18 @@
+drop TABLE if EXISTS dbo.CleaningCompany
+go
+CREATE table dbo.CleaningCompany(
+        CustomerID int not null IDENTITY primary key,
+        FirstName varchar(50) not null constraint ck_Customer_First_name_is_not_blank CHECK (FirstName <> ''), 
+        LastName varchar(50) not null constraint ck_Customer_Last_name_is_not_blank CHECK (LastName <> ''), 
+        PhoneNumber varchar(15) not null constraint ck_Customer_Phone_Number_is_not_blank CHECK (PhoneNumber <> ''), 
+        CustomerAddress VARCHAR(50) not null constraint ck_Customer_Address_is_not_blank CHECK (CustomerAddress <> ''), 
+        Frequency VARCHAR(9) not null  constraint ck_Customer_frequency_must_be_either_weekly_or_biweekly check (frequency in ('Weekly', 'BiWeekly')), 
+        PricePerHour int not null CONSTRAINT ck_Customer_PricePerHour_is_between_0_and_53 check (PricePerHour between 0 and 53),
+        TimeinHours int not null CONSTRAINT ck_Customer_TimeinHours_is_between_greater_than_0 check (TimeinHours >0 ),
```

Comment:
This should have some sort of decimal place as this should be the avg hours and it can be 1.5 hours.

Reply:
switched the datatype to decimal